packages/web-worker/README.md

```diff
@@ -65,6 +65,22 @@ const result = await worker.default(
 ); // 'Hello, Tobi'
 ```
 
+When the web worker is no longer required, you can terminate the worker using `terminate()`. This will immedietly terminate any ongoing operations. If an attempt is made to make calls to a terminated worker, an error will be thrown.
```

Comment:
```suggestion
When the web worker is no longer required, you can terminate the worker using `terminate()`. This will immediately terminate any ongoing operations. If an attempt is made to make calls to a terminated worker, an error will be thrown.
```